docs/Introduction-QuickStartGuide.md

```diff
@@ -568,6 +568,14 @@ export default {commit};
 
 In the simplest case above, we just need to pass an `optimisticResponse` option, which should refer to an object having the same shape as the mutation response payload. When we pass this option, Relay will know to immediately update our local data with the optimistic response, and then update it with the actual server response or roll it back if an error occurs.
 
+Please note that the actual query and response payload may not have the exact same shape as the selection in your code, because sometimes Relay will select extra fields for you. For example:
```

Comment:
Do you think it would make sense to explicitly state that this means people need to add these fields to their optimistic response?